contrib/jms/README.md

```diff
@@ -0,0 +1,45 @@
+# JMS module
```

Comment:
probably you want to say what JMS is and provide a link somewhere in the first line or two of the README